common/src/main/java/com/github/khanshoaib3/minecraft_access/features/point_of_interest/LockingHandler.java

```diff
@@ -161,8 +160,7 @@ private void mainLogic() {
                     PlayerUtils.playSoundOnPlayer(SoundEvents.BLOCK_NOTE_BLOCK_PLING, aimAssistAudioCuesVolume, bowState);
                     lastAimAssistCue = 1;
                 }
-            }
-            else {
+            } else {
                 if (lastAimAssistCue != 0 || bowState != lastBowState) {
```

Comment:
Should this not be an else if statement?
```suggestion
            } else if (lastAimAssistCue != 0 || bowState != lastBowState) {
```

Reply:
This class is complication, I already refactored it but it's still complicated, @Molitvan 's bow aiming assist added more complexity. That's why I want to recode the POI and locking system.